DPRHTML/index.html

```diff
@@ -7,11 +7,11 @@
 
     <title>Digital Pali Reader (alpha)</title>
 
-    <link rel="stylesheet" href="/digitalpalireader/content/js/external/bootstrap/4.4.1/css/bootstrap.min.css" />
-    <link rel="stylesheet" href="/digitalpalireader/content/js/external/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="/digitalpalireader/content/js/external/bootstrap/4.4.1/css/bootstrap.min.css">
```

Comment:
Oh no. This will break a lot of stuff. I believe it was an error in resolution of conflicts from merge from master in your fork.

Reply:
ok, corrected